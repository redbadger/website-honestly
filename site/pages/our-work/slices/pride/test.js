import React from 'react';
import { shallow } from 'enzyme';

import PrideSlice from './index';

describe('pages/our-work/case-study/slices/pride', () => {
  it('renders correctly', () => {
    const rendered = shallow(<PrideSlice />);
    expect(rendered).toMatchSnapshot();
  });
});
