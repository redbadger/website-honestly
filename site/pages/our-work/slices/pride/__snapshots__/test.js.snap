// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/our-work/case-study/slices/pride renders correctly 1`] = `
<Container
  layout="row-reverse"
  to="prideCaseStudy"
>
  <Text
    layout="rightCol"
  >
    <Header>
      Delivering a best-in-class digital experience
    </Header>
    <Description>
      Building a 5 star digital experience for Pride in Londonâ€™s mobile app using React Native, promoting inclusion and accessibility for an improved user experience.
    </Description>
    <ReadMore />
  </Text>
  <Image
    alt="Pride heart animation"
    src="listing.gif"
  />
</Container>
`;
